features/in-app-install.yml

```diff
@@ -0,0 +1,12 @@
+name: In-app install experience
+description: On browsers that support installing Progressive Web Apps (PWAs), the `beforeinstallprompt` event can be used to defer the browser installation flow until when the user interacts with an in-app install button.
```

Comment:
```suggestion
description: The `beforeinstallprompt` event fires when a Progressive Web App (PWA) is installable, typically on page load in browsers that support installing PWAs. You can use this event to ask users to install within your site, instead of using a generic prompt provided by the browser.
```

Reply:
Thanks, I'll use this but tweak it a little bit, mostly because the in-app install option doesn't "replace" the browser prompt altogether. It delays it until the user interacts with whatever in-app UI you're displaying.